orttraining/orttraining/python/training/ortmodule/_custom_autograd_function_exporter.py

```diff
@@ -37,7 +37,8 @@ def _export_pt_1_10(g, n, *args, **kwargs):
                 "wrap exportable sub-nn.Module's as ORTModule."
             )
         inplace = kwargs["inplace"]
-        training_mode = symbolic_helper._training_mode
+        # TODO move to public API once exporter team exposes that
+        training_mode = _globals.GLOBALS.training_mode
```

Comment:
just confirm, is this compatible with older version PyTorch? If not, should we add torch version check here?

Reply:
It is recently introduced. I would just extract it into a function (something like `get_exporter_training_mode`) and do a try catch there. There is a risk that we will need to handle arbitrarily many different cases to access this var because internal things can change many times in nightly builds.